test/bigtcp/test.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
You might want to run this test as part of the CI L4LB. To make it happen, you need to add exec the test.sh here: https://github.com/cilium/cilium/blob/master/.github/workflows/tests-l4lb.yaml#L198

Reply:
ah nice catch, I'll update the test commit to include it, thanks!